### ISSUE TITLE
Add setting so version is only incremented once

### ISSUE THREAD

**Issue by grbsk** [semantic-gitlog/maven-semantic-gitlog#15]
Currently, it seems that each non-hidden commit type it finds increments the new version by one. So if I release version 0.0.1, then I add 3 `fix:` commits, the new suggested version is 0.0.4. It works similarly for `feat` with the minor version, and I would venture a guess with breaking/major. 

Maybe this is desirable for some people, but it would be great if we could add a setting (if it doesn't already exist) so that once it determines what part of the version it needs to bump, it just increments by one.

Here is my example `build` in pom:

```
   <build>
        <plugins>
            <plugin>
                <groupId>team.yi.maven.plugin</groupId>
                <artifactId>maven-semantic-gitlog</artifactId>
                <version>0.22.0</version>
                <configuration>
                    <fileSets>
                        <fileSet>
                            <target>${project.basedir}/CHANGELOG.md</target>
                            <template>${project.basedir}/config/CHANGELOG.mustache.md</template>
                        </fileSet>
                    </fileSets>

                    <updateProjectVersion>false</updateProjectVersion>
                    <derivedVersionMark>NEXT_VERSION:==</derivedVersionMark>
                </configuration>
            </plugin>
        </plugins>
    </build>
```

**Comment by ococl** [comment]
I'm so sorry I have no time to reply to you! It's too busy these days.

**about your issue, I have these suggestions:**

> this doc maybe help you: https://semantic-gitlog.github.io/semantic-gitlog/#/en-us/components/semantic-gitlog/?id=configuration

- `isUnstable` It is useful to suppress the version surge caused by breaking changes during the development phase.
- `strategy` if set `slow`, Regardless of how many similar changes there are, the corresponding version number is increased only once.
- You can also use some options to customize the relationship between the commit type and the version parts: `majorTypes`, `minorTypes`, `patchTypes` and `preReleaseTypes`, `buildMetaDataTypes`.

the code logic at here: https://github.com/semantic-gitlog/semantic-gitlog/blob/master/src/main/java/team/yi/tools/semanticgitlog/VersionDeriver.java#L36

**Comment by grbsk** [comment]
Awesome, I had a feeling i just needed a setting. `strategy="slow"` looks like the winning ticket. Thanks for the library!

**Comment by grbsk** [comment]
Oops, a little premature in closing it. Looks like "strategy" isn't exposed to the plugin configuration so I can't set it in pom.xml.

**Comment by ococl** [comment]
Well, I compared my notes, and I removed the option "strategy" before publishing the code.

I'll add it again in few days.

**Comment by ococl** [comment]
I released version [0.22.1](https://search.maven.org/artifact/team.yi.maven.plugin/maven-semantic-gitlog), please update it.

**Comment by grbsk** [comment]
Hi! Thanks for the update. I am seeing better effect with `strategy="slow"`. There is one unexpected behavior though, and that is when I have a breaking change AND either a fix or feat. But I think that's an issue in the underlying gitlog library and not the maven plug-in, so I'll close this and open there. Thanks!
